homeassistant/components/flux_led/light.py

```diff
@@ -381,16 +381,20 @@ def white_value(self):
         """Return the white value of this light."""
         return self._white_value
 
+    def get_bulb_rgbww(self):
+        """Return the getRgbww value from the bulb."""
+        return self._bulb.getRgbww()
+
     @property
```

Comment:
I meant that these properties should not be properties but methods.
```suggestion
```

Reply:
Gotcha - fixed